### ISSUE TITLE
Placeholder for inputs

### ISSUE THREAD

**Issue by vitoraraujodev** [arunghosh/react-pin-input#191]
Hello devs!
I was wondering how to set placeholders for every input, as the param "placehoder" doesn't exist.
Actually, I managed to do this with javascript, but it's not the best solution cuz when you text inside the input and delete it, the placeholder isn't there anymore:

    const [inputs, setInputs] = useState([]);

    useEffect(() => {
        setInputs(document.getElementsByClassName('pincode-input-text'));
        for (let iinput = 0; input < inputs.length; input += 1) {
            inputs[i].setAttribute('placeholder', '•');
        }
    }, [inputs]);

Any suggestions?

**Comment by IvanTanaka** [comment]
anyone solved this yet? I don't find the placeholder too, even though in closed issue #100 shown that placeholder is available, but I can't find it. Please help

**Comment by arunghosh** [comment]
Seems it is not implemented correctly. The `placeholder` and `aria-label` syncs with `value`. Not sure if that is what you need.
```
      <input
        .....
        placeholder={value}
        aria-label={value}
        .......
        value={value}
      />
 ```

**Comment by rapolaslisboa** [comment]
@arunghosh would it be possible to publish a new version of the package with custom placeholders / aria-labels? I've seen you merged Johnny's PR but haven't published a new version

**Comment by arunghosh** [comment]
@rapolaslisboa Sorry for the delay. I will publish it soon.

**Comment by arunghosh** [comment]
@rapolaslisboa It is published now in `v1.1.0`
